README.md

```diff
@@ -1,7 +1,6 @@
 # All Contributors v1.0.0-beta.0
 
-This is a specification for recognizing contributors to an open source project in a way that rewards any and every contribution
-whether or not it be code.
+This is a specification for recognizing contributors to an open source project in a way that rewards any and every contribution, not just code.
```

Comment:
I'd almost agree, but "any" sounds more inclusive. On the other hand, using "each" makes it sound more important. It's a toss-up ;)